pkgs/development/web/postman/default.nix

```diff
@@ -2,11 +2,11 @@
 
 stdenv.mkDerivation rec {
   name = "postman-${version}";
-  version = "6.3.0";
+  version = "6.7.1";
 
   src = fetchurl {
     url = "https://dl.pstmn.io/download/version/${version}/linux64";
-    sha256 = "09m511y977478567lc28mhy68b99ssajzhirc1c4anxnvvs7s6fa";
+    sha256 = "091k51km4lfxlksm8imkr30yli0zzbbajqp0fdk7r6migzm5k960";
```

Comment:
I get a different checksum here:

```
hash mismatch in fixed-output derivation '/nix/store/b5lzk1wa0087msx6316wsflfpihr2p24-postman-6.7.1.tar.gz':
  wanted: sha256:091k51km4lfxlksm8imkr30yli0zzbbajqp0fdk7r6migzm5k960
  got:    sha256:1x8jj0xs67wi0qj6x22h54crndml6fl8a128s57v058fyxji6brx
cannot build derivation '/nix/store/rawmx5sbnnam3dr0542c7vbfj0hqz7qr-postman-6.7.1.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/mi4r8llng56aw0ma4qqhz4pvnb755rsx-env.drv': 1 dependencies couldn't be built
```

Reply:
Now I am as well, I wonder why it worked last night with the different sha sum...
will fix.